tests/data/test_boyd_study/extract_configs/manifest_180917_rectified_map.py

```diff
@@ -12,6 +12,10 @@
 source_data_loading_parameters = {}
 
 operations = [
+    keep_map(
+        in_col='Library',
+        out_col=CONCEPT.SEQUENCING.ID
```

Comment:
We shouldn't have to extract this. The library name is already a unique identifier for a sequencing experiment unless otherwise specified (for a multi-library) sequence row.

Reply:
Same as above